jumpscale/packages/admin/actors/admin.py

```diff
@@ -146,6 +146,11 @@ def get_config(self) -> str:
         config_obj = j.core.config.get_config()
         return j.data.serializers.json.dumps({"data": config_obj})
 
+    @actor_method
+    def get_sdk_version(self) -> str:
+        sdk_version = j.core.config.get_current_version()
```

Comment:
I think that's the ng version not the sdk version ?

Reply:
let's track both